docs/reference.md

```diff
@@ -966,6 +983,195 @@ users from seeing authenticated pages momentarily during page load. See the
 ["Using Authentication with Routers"](/docs/guide/user-auth.md#authenticating-with-routers)
 section of our AngularFire guide for more information and a full example.
 
+## $firebaseStorage
+
+AngularFire includes support for [binary storage](/docs/guide/uploading-downloading-binary-content.md)
+with the `$firebaseStorage` service.
+
+The `$firebaseStorage` service takes a Firebase Storage reference.
+
+```js
+app.controller("MyCtrl", ["$scope", "$firebaseStorage",
+  function($scope, $firebaseStorage) {
+    var storageRef = firebase.storage().ref("images/dog");
+    $scope.storage = $firebaseAuth(storageRef);
+  }
+]);
+```
+
+The storage object returned by `$firebaseStorage` contains several methods for uploading and
+downloading binary content, as well as managing the content's metadata.
+
+### $put(file, metadata)
+
+[Uploads a `Blob` object](https://firebase.google.com/docs/storage/web/upload-files) to the specified storage reference's path with an optional metadata parameter.
+Returns an [UploadTask](#upload-task) wrapped by AngularFire.
+
+
+```js
+var htmlFile = new Blob(['<html></html>'], {type : 'text/html'});
+var uploadTask = $scope.storage.$put(htmlFile, { contentType: 'text/html' });
+```
+
+## $putString(string, format, metadata)
+
+[Uploads a RAW, base64 string, or base64 URL string](https://firebase.google.com/docs/storage/web/upload-files#upload_from_a_string) to the specified storage reference's path with an optional metadata parameter.
+Returns an [UploadTask](#upload-task) wrapped by AngularFire.
+
+```js
+var base64String = '5b6p5Y+344GX44G+44GX44Gf77yB44GK44KB44Gn44Go44GG77yB';
+var uploadTask = $scope.storage.$putString(base64String, 'base64', { contentType: 'image/gif' });
+```
+
```

Comment:
Let's add a note here that valid values for `format` are `"raw"`, `"base64"`, `"base64url"`, and `"data_url"`.